concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1815,8 +1815,12 @@ jobs:
           file: paas-cf/concourse/tasks/smoke-tests-run.yml
 
         ensure:
-          task: remove-temp-user
-          file: paas-cf/concourse/tasks/delete_admin.yml
+          do:
+          - task: get-cf-cli-config
+            file: paas-cf/concourse/tasks/get-cf-cli-config.yml
+
```

Comment:
I think you should merge get-cf-cli-config into remove user:
* to avoid having to declare it multiple times like you do
* to avoid concourse having to launch more containers
* to have simpler, cleaner pipelines